modules/ossm-tutorial-jaeger-generating-traces.adoc

```diff
@@ -33,8 +33,13 @@ This tutorial uses {ProductShortName} and the Bookinfo tutorial to demonstrate h
 .. In the {product-title} console, navigate to *Networking* -> *Routes* and search for the Jaeger route, which is the URL listed under *Location*.
 .. Use the CLI to query for details of the route:
 +
+[NOTE]
+====
+Replace `<control_plane_project>` with the name of your control plane project. In this example, the control plane project is `istio-system`.
+====
```

Comment:
@jwennerberg in the example you have replaced `istio-system` with `<control_plane_project>`
But still, referring to `istio-system` in the note.

we have to either add `istio-system` in the example or remove `istio-system` from the note.

@JStickler your suggestion, please.

Reply:
Now that I have time to think about it, we put "istio-system" in the command specifically so that users could use the new copy feature to copy the command from the docs.  The note is there in case they were using a different namespace from the tutorial instructions.